image_classification/tensorflow/official/resnet/resnet_run_loop.py

```diff
@@ -401,12 +429,30 @@ def resnet_main(seed, flags, model_function, input_function, shape=None):
   else:
     benchmark_logger = None
 
-  for _ in range(flags.train_epochs // flags.epochs_between_evals):
+  mlperf_log.resnet_print(key=mlperf_log.TRAIN_LOOP)
+  success = False
+  for i in range(flags.train_epochs // flags.epochs_between_evals):
+    for j in range(flags.epochs_between_evals):
+      mlperf_log.resnet_print(key=mlperf_log.TRAIN_EPOCH,
+                              value=i * flags.epochs_between_evals + j)
     train_hooks = hooks_helper.get_train_hooks(
         flags.hooks,
         batch_size=flags.batch_size,
         benchmark_log_dir=flags.benchmark_log_dir)
 
+    _log_cache = []
+    def formatter(x):
+      """Abuse side effects to get tensors out of the model_fn."""
+      if _log_cache: _log_cache.pop()
```

Comment:
I don't like if statements that are all on one line.

Reply:
We all don't like things. But I happen to be in a charitable mood.